pkg/action/scan.go

```diff
@@ -58,6 +58,8 @@ func findFilesRecursively(ctx context.Context, root string, c Config) ([]string,
 
 // cleanPath removes the temporary directory prefix from the path.
 func cleanPath(path string, prefix string) string {
+	// if the path has "private" prefix, remove it
+	path = strings.ReplaceAll(path, "/private", "")
```

Comment:
Not sure why this started happening (on MacOS) but the `path` had a `/private/...` prefix which caused the `TrimPrefix` to return the original path unaltered.